docs/supported_apis/series_supported.rst

```diff
@@ -12,13 +12,16 @@ the method in the left column. ``Y`` stands for yes, ``N`` stands for no, ``P``
 for partial (meaning some parameters may not be supported yet), and ``D`` stands for
 default to pandas. To learn more about the implementations that default to pandas, see
 the related section on :doc:`Defaulting to pandas </supported_apis/index>`.
+**Currently second column reflects implementation status for ``Ray`` and ``Dask`` engines. By default support of a method
+in the ``Omnisci`` engine could be treated as ``D`` unless ``Notes`` column contains additional information. Similarly,
+by default ``Notes`` contains information about ``Ray`` and ``Dask`` engines unless ``Omnisci`` explicitly mentioned.**
 
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | Series method               | Modin Implementation? (Y/N/P/D) | Notes for Current implementation                   |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``abs``                     | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``add``                     | Y                               |                                                    |
+| ``add``                     | Y                               | **Omnisci**:``Y``                                  |

```

Comment:
I wonder if we should mention this case for bin ops in the notes:
https://github.com/modin-project/modin/blob/95d71b2bde0ceb40cb1b64fa762b3009a94168dd/modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/dataframe.py#L1245-L1252

Reply:
Is this true for Series also?

Reply:
> Is this true for Series also?

Yes it is:

```python
>>> import modin.pandas as pd
>>> sr1 = pd.Series([1, 2, 3, 4])
>>> sr2 = pd.Series([10, 20, 30, 40])
>>> sr1 + sr2 # Using two independent Series -> getting a warning
UserWarning: unsupported binary op args (outer join is not supported) defaulting to pandas implementation.
Please refer to https://modin.readthedocs.io/en/stable/supported_apis/defaulting_to_pandas.html for explanation.
UserWarning: <function DataFrame.add> defaulting to pandas implementation.
0    11
1    22
2    33
3    44
dtype: int64
>>> sr1 + sr1 # Using two identical Series -> getting no warning
0    2
1    4
2    6
3    8
dtype: int64
```